tests/PHPStan/Analyser/data/http-response-header.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+use function PHPStan\Testing\assertType;
+
+file_get_contents('https://example.org');
+assertType('list<string>', $http_response_header);
```

Comment:
ohh I think you are missing another assert for `assertNativeType`

Reply:
Ok hopefully I did this correctly :)